examples/mysql/README.md

```diff
@@ -0,0 +1,48 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
Missed this in the other PR, but could you add a small introduction like https://github.com/humanitec-architecture/resource-packs-aws/tree/main/examples/mysql/aurora#example-mysql-resource-based-on-aws-rds-aurora?

Reply:
Added!